packages/apps/docs/src/docs/reference/chainweb/chainweb-api.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Chainweb REST API
+description:
+  The Chainweb node API consists of two distinct sets of API endpoints. The peer-to-peer (P2P) API provides endpoints for nodes to communicate with each other. The Chainweb service API provides endpoints for clients interacting with nodes.
+menu: Chainweb API
+label: Introduction
+order: 2
+layout: full
+tags: ['chainweb-node', 'rest api', 'chainweb api', 'node operators']
+---
+
+# Chainweb node API
+
+As discussed in [Chainweb simplified](/learn/chainweb), Chainweb is a scalable proof-of-work (PoW) consensus algorithm. 
+Chainweb extends the Bitcoin Nakamoto consensus algorithm, a protocol designed for a single chain, to handle consensus with transactions coming from multiple parallel chains. 
+Because Chainweb scales horizontally by using multiple chains, the Kadena network provides better transaction throughput with lower transaction fees than other proof-of-work blockchains.
+
+The Chainweb node API consists of two distinct APIs: the peer-to-peer (P2P) API and the service API. 
+
+- The **peer-to-peer API** contains the RPC endpoints that nodes must expose to support consensus and the queueing and processing of transactions. The peer-to-peer endpoints are intended primarily for communication between Chainweb nodes and are typically more efficient than the service API endpoints for most use cases. 
+
+- The **service API** contains endpoints that can be used to connect to Chainweb nodes from clients outside of the blockchain network.
+How you connect to the endpoints in these two APIs depends on the configuration of the Chainweb node.
+
+For more information about the peer-to-peer API and its endpoints, see [Peer-to-peer API](#peer-to-peer-api).
+For more information about the service API and its endpoints, see [Service API](#service-api).
+
+## Getting documentation
+
+This part of the documentation summarizes the key endpoints you can use to query and update Chainweb nodes and the data models you should be familiar with when interacting with Chainweb nodes and the blockchain network.
+
+For full documentation of the Chainweb node API, including sample requests and responses, see the generated [Kadena Chainweb Node OpenAPI](https://api.chainweb.com/openapi/).
+You can also build and serve the OpenAPI documentation locally from a Docker container.
+
+To build a local copy of the documentation:
+
+1. Open a terminal shell on your computer.
+
+2. Clone the `chainweb-openapi` repository by running the following command:
```

Comment:
I actually think it would make a lot of sense, but only if the team really wants to maintain the OpenAPI spec. 
Auto-generated documentation is only really useful if it is kept up to date consistently. Also, it should really follow some styling/branding so that it doesn't look completely out of step with all other Kadena-related documentation.
Personally, I'm not a fan of the styling with all of the boxes and buttons and nested collapsible parameters inside of tables. I like things really simple with minimal distracting gadgetry.